controller/Session.class.php

```diff
@@ -31,6 +34,18 @@ public static function init()
             static::setNamespace(static::NAMESPACE_DEFAULT, $oldSession);
         }
 
+        Response::addPostRenderCallback(function(){
+            $site_visitor_id = key_exists(static::USER_ID, $_SESSION) ? $_SESSION[static::USER_ID] : '';
```

Comment:
This is fine, but in "modern" PHP, a more common way to write it would be:

`$_SESSION[static::USER_ID] ?? ''`

`??` is a null coalesce operator.

Reply:
Thanks! Keep these types of things coming. I am basically googling syntax for what I want to do and sometimes SO is out of date.